vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -2374,6 +2385,29 @@ class TimelineFragment @Inject constructor(
         }
     }
 
+    /**
+     * Display a dialog that will let the user to enable threads
+     */
+
+    private fun displayThreadsBetaNotice() =
```

Comment:
I do not like using `=` when the fun should not return anything. Can you use `{` instead?

Here it will return a `TextView?` which is highly unexpected.

<img width="387" alt="image" src="https://user-images.githubusercontent.com/3940906/161733567-c09a2707-98a6-4de9-9634-21d018308083.png">

Reply:
I agree, and I prefer that than return Unit manually